track_history/tracker.py

```diff
@@ -117,7 +132,10 @@ def signal_receiver(self, instance, signal, **kwargs):
             DJANGO_SAFEDELETE_INSTALLED
             and signal == post_save
             and not created
-            and kwargs.get("update_fields", []) == ["deleted"]
+            and (
+                kwargs.get("update_fields", [])
+                == ["deleted"]
+            )
```

Comment:
Q: Did black or something reformatted it? Previous version looked quite ok 🤔

Reply:
Not sure, fixed :P